src/test/java/TGHW25.java

```diff
@@ -0,0 +1,25 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class TGHW25 extends BaseTest{
+    @Test(dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
+    public static void loginEmptyEmailPasswordTest (String email, String password) {
+        login(email, password);
+        Assert.assertEquals(getDriver().getCurrentUrl(), url);
+    }
+    @Test(enabled = true, priority = 1, description = "LoginValidEmailValidPasswordTest")
+    public static void loginValidEmailValidPasswordTest () throws InterruptedException {
+        HomePage homePage = new HomePage(getDriver());
+        LoginPage loginPage = new LoginPage(getDriver());
+
+        loginPage.logIn();
+        homePage.getUserAvatar();
+
+        Thread.sleep(5000);
```

Comment:
Please, delete ` Thread.sleep(5000);`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.


![image](https://user-images.githubusercontent.com/104761482/218298619-32a6e4d5-4d3b-466e-8824-199de3d4697c.png)